environments/backfillEvents.toml

```diff
@@ -28,43 +28,60 @@
         type = "eth_event"
         repository = "github.com/makerdao/vdb-mcd-transformers"
         migrations = "db/migrations"
-        contracts = ["MCD_FLAP", "MCD_FLIP_BAT_A", "MCD_FLIP_ETH_A", "MCD_FLIP_KNC_A", "MCD_FLIP_SAI",
-                     "MCD_FLIP_TUSD_A", "MCD_FLIP_USDC_A", "MCD_FLIP_USDC_B", "MCD_FLIP_WBTC_A", "MCD_FLIP_ZRX_A",
-                     "MCD_FLOP"]
+        contracts = ["MCD_FLAP_1.0.0", "MCD_FLAP_1.0.9", "MCD_FLIP_BAT_A_1.0.0", "MCD_FLIP_BAT_A_1.0.9",
```

Comment:
Yeah hoping we don't need to backfill, so could wipe this file - was mostly just adding compatibility (the version of this config currently on `breaking-changes` breaks the build for this PR since the directories are no longer recognized)